libbitstream/CMakeLists.txt

```diff
@@ -0,0 +1,69 @@
+## Copyright(c) 2019, Intel Corporation
```

Comment:
Are we adding this bitstream as a separate shared library which was previously part of xfpga library?

Reply:
This is a merge from master.